pyswagger/primitives/codec.py

```diff
@@ -0,0 +1,53 @@
+import json
+import six
+from .comm import PrimJSONEncoder
+
+
+class MimeCodec:
+    def __init__(self):
+        self._codecs = {}
+        self.register('text/plain', PlainCodec())
+        jsonCodec = JsonCodec()
+        for _type in ['application', 'text']:
```

Comment:
IMO, keywords are better kept in one string than separated into parts. Could you rewrite this part in this form?
```python
self.register('application/json', jsonCodec)
self.register('text/json', jsonCodec)
```
In this form, it's easier to trace code for new comers.